modules/virt-edit-cdrom-vm.adoc

```diff
@@ -9,37 +9,33 @@
 ifeval::["{context}" == "virt-edit-vms"]
 :object: virtual machine
 :object-gui: Virtual Machines
+:object-overview: Virtual Machine Overview
 endif::[]
 
 ifeval::["{context}" == "virt-editing-vm-template"]
 :object: virtual machine template
 :object-gui: Templates
+:object-overview: VM Template Details
 endif::[]
 
 [id="virt-vm-edit-cdrom_{context}"]
 
 = Editing CD-ROMs for {object-gui}
-Use the following procedure to configure CD-ROMs for virtual machines.
+Use the following procedure to edit CD-ROMs for {object}s.
 
 .Procedure
 
-. From the *{object-gui}* tab, select your {object}.
+. Click *Workloads* -> *Virtualization* from the side menu.
 
-. Select the *Overview* tab.
+. Click the *{object-gui}* tab to open the *{object-overview}* screen.
```

Comment:
The steps should be:
`. Click the *{object-gui}* tab`. 
`. Select a {object} to open the *{object-overview}* screen.`
`. Click the *Disks* tab.`

Attaching rendered output for VMs and Templates for reference.

![image](https://user-images.githubusercontent.com/64486364/107578270-88a05080-6bc1-11eb-8a04-9fd36bc1da3e.png)

![image](https://user-images.githubusercontent.com/64486364/107578393-ae2d5a00-6bc1-11eb-95e1-033165a897c4.png)

Reply:
This one definitely a victim of the GitHub outage! The changes never rippled through to show up in GitHub...

Reply:
Verified correct rendering:
https://deploy-preview-29092--osdocs.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-editing-vm-template.html#virt-vm-edit-cdrom_virt-editing-vm-template

https://deploy-preview-29092--osdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-edit-vms.html#virt-vm-edit-cdrom_virt-edit-vms